include/nn/bcat.h

```diff
@@ -0,0 +1,109 @@
+/**
```

Comment:
Please run clang-format on this file to ensure the code style is consistent with the rest of the headers.

Reply:
Thanks for the suggestions! I fixed a ton of things but when I try to run clang format I get an error syaing at line 10 there's an invalid boolean, am I doing something wrong? (I just did clang-format -i include/nn/bcat.h)

Reply:
Actually I think instead of "Never" there should be "false"

Reply:
Yeah I think never is only compatible with the latest version of clang-format. Feel free to change it to false to fix compatibility with older versions